fitbenchmarking/cli/main.py

```diff
@@ -81,12 +81,72 @@ def get_parser():
     parser.add_argument('-d', '--debug-mode',
                         default=False,
                         action='store_true',
-                        help='Enable debug mode (prints traceback)',)
-
+                        help='Enable debug mode (prints traceback).',)
+    parser.add_argument('-n','--num_runs',
+                        metavar='NUM_RUNS',
+                        type = int,
+                        default = 0,
+                        help = "Set the number of runs to average each fit over.")
+    parser.add_argument('-a', '--algorithm_type',
+                        metavar = 'ALGORITHM_TYPE',
+                        nargs='+',
+                        type = str,
+                        default = [],
+                        help = "Select what type of algorithm is used within a specific software.")
```

Comment:
The style checker is also upset about line length here, we can just break it up and it should be happy.
```suggestion
                        help="Select what type of algorithm is used within a "
                             "specific software.")
```